src/feeds/FeedReader.ts

```diff
@@ -55,6 +55,10 @@ export interface FeedEntry {
     fetchKey: string,
 }
 
+export interface FeedSuccess {
+    url: string;
```

Comment:
Nitpick, should this end with a colon instead?

Reply:
comma? probably.